flexget/plugins/modify/path_select.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import os
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+
+log = logging.getLogger('path_select')
+
+
+def percentage(part, whole):
+    try:
+        return 100 * float(part)/float(whole)
```

Comment:
Doesn't importing division from future mean we don't have to explicitly turn these in to floats?